test/libsolidity/syntaxTests/inline_arrays/unnamed_types_in_inline_array_1.sol

```diff
@@ -4,4 +4,4 @@ contract C {
     }
 }
 // ----
-// TypeError 9656: (47-52): Unable to deduce nameable type for array elements. Try adding explicit type conversion for the first element.
+// TypeError 9563: (48-51): Invalid mobile type.
```

Comment:
The error message can be improved in a later PR.

Reply:
Is there an open issue for that? So, we don't forget to do it later.

Reply:
Not sure we need one. This is one of the things that will change when we rewrite analysis and we likely won't be addressing it in isolation. The whole concept of a "mobile type" may be gone, along with implicit conversions, at least for non-literals.

Reply:
Just noticed that I had opened an issue for it when reviewing https://github.com/ethereum/solidity/pull/14347, but I closed it now as not planned :)